plugins/inputs/prometheus/prometheus.go

```diff
@@ -163,6 +163,9 @@ func (p *Prometheus) Init() error {
 	}
 
 	ctx := context.Background()
+	// The parameter should be replaced in future with a new name called timeout to directly parse the configuration.
+	// I've decided to handle it not directly because, it's a breaking change.
```

Comment:
Like to avoid TODO comments, I would either deprecate the option now, or probably remove the comments as this will be good going forward.

Reply:
I'll remove it and place the topic in a separate GitHub issue.